ui/src/client/Clients.tsx

```diff
@@ -119,6 +123,7 @@ const Row: SFC<IRowProps> = ({name, value, fEdit, fDelete}) => (
                 style={{display: 'flex', alignItems: 'center', width: 200}}
             />
         </TableCell>
+        <TableCell>{lastSeen ? <TimeAgo date={lastSeen} /> : 'Never'}</TableCell>
```

Comment:
Maybe we can do something like if (now - lastSeen) < 5min then it only displays the text "recently".